### DISCUSSION TITLE
In Kanban View, I want to click on the filename to go directly to the Note.

### DISCUSSION THREAD

**Discussion by virtuabart** [marcusolsson/obsidian-projects#695] [Help] [answered]
Since there are 2 names when you enable the field, Name, on my Project. I hid this via Css so I can click directly on the name to edit the note.

![image](https://github.com/marcusolsson/obsidian-projects/assets/51690385/2b6bca24-37e6-4075-ab8e-9a47fdd6bc42)

Is this the right way?

**Comment by virtuabart** [comment]
The CSS.

.card-header.svelte-7vepl5 {
	display: none;
}

Works like a charm for me, now there is only one filename and it is clickable to go directly inside the Note.

**Comment by virtuabart** [comment]
![image](https://github.com/marcusolsson/obsidian-projects/assets/51690385/c8871c90-06ca-408d-839a-6e16c57c9dab)

**Comment by Acylation** [comment, marked as answer]
In the plugin’s settings tab, there is an option to select the default behavior when clicking on a link. You can choose between opening a note or opening the edit modal. With the Ctrl/Cmd key it is expected to perform the opposite.

FYI #602 fixed by #620

**Reply by virtuabart** [reply]
You are amazing! Thank you so much for "pay"ing attention to my question. God bless.
